tests/Tests/500__jupyterhub/minimal-pytorch-test.robot

```diff
@@ -34,6 +34,19 @@ PyTorch Image Workload Test
     Capture Page Screenshot
     JupyterLab Code Cell Error Output Should Not Be Visible
 
+Tensorboard Workload Test
+    [Documentation]  Verifies that tensorboard is accessible
+    [Tags]  Sanity
+    ...     PLACEHOLDER
```

Comment:
can "placeholder" be removed or is it here for a specific reason?

Reply:
Placeholder is there for the `Category` tags, which I think we've never implemented